skietbaan_fe/src/reducers/postReducer.js

```diff
@@ -0,0 +1,28 @@
+import {FETCH_POSTS, NEW_POST} from '../actions/types';
+
+const initialState = {
+    items: [],
```

Comment:
`items[]` is an array of `item{}`? if so, then why do you have it there? or the `item{}` is like a selected item? 

i suggest something like

```
allItems: [],
selectedItem: {}
```

For reviewers to understand the difference in the two

Reply:
Thanks for a constructive advise. Regards